test/library/memory-store-test.ts

```diff
@@ -132,4 +132,16 @@ describe('memory store test', () => {
 			clearTimeout(realTimeoutId)
 		}
 	})
+
+	it('When calling init it should reset interval when already initialized', async () => {
+		const shutdownSpy = jest.spyOn(MemoryStore.prototype, 'shutdown')
```

Comment:
Is the `.prototype` here necessary? If not, could you please drop it?

IIRC we should just need to pass the class to the `spyOn` function, no need to mess with prototype.